lib/source/search/index.html.erb

```diff
@@ -0,0 +1,26 @@
+---
+title: Search
+index: false
+prevent_indexing: true
+hide_in_navigation: true
+---
+
+<div id="search-help" class="search-help">
+  <%# strange id is to avoid clash with search input id #%>
+  <h1 id="how-to-search">Search</h1>
+
+  <p class="govuk-body search-help__no-js">
+    This page requires JavaScript to perform searches. Please enable JavaScript, or use the search input to search using Google.
```

Comment:
```suggestion
You need to turn on JavaScript in your browser to search this site. You can either:

<ul>
    <li>turn on JavaScript in your browser</li>
    <li>reenter your search term and select **Search** again, to search this site using Google</li>
</ul>
```

Reply:
Do you want 'Search' to be bold, or surrounded by asterisks?

Reply:
Ah oops, I was in Markdown mode. Yep that should be bold, sorry.

Reply:
Hmm, I'm not sure about 'reenter... and select Search again'; in most cases readers should only see this content if they've not entered a search term.

(They could also reach this content if they visit the search page with a search query already in the URL, but that seems unlikely to me).

Reply:
Ah - I thought they would see the "This page requires JavaScript..." paragraph only if they searched with JS off? When I search without a search term, I get the other paragraph ("Search the documentation using the search input.")

Reply:
Sorry, maybe I should have made it clearer in the search commit or somewhere else... my thinking for this content (both content about JavaScript and without) is that it would be there in case the user somehow found the search page without entering a search query; the most common way I think this would happen is by clicking the search button without entering anything into the search box. If they had entered a search term they wouldn't need the help text, was my thinking.

I'm happy to have a call to discuss further, if that would be helpful.

Reply:
Thanks for taking me through this. So if I've understood right, I think this would work for the 'first' paragraph?

```
You need to turn on JavaScript in your browser to search this site. You can either:

<ul>
    <li>turn on JavaScript in your browser</li>
    <li>search this site using Google</li>
</ul>
```

Then (if they searched for nothing AND JS is off) they'll also see the 'second' paragraph, which would be:

```
Enter a search term in the search box, then select the search button.
```

Let me know if I got anything wrong!

Reply:
Yep, that looks good!

I've made those changes now, please take another look to check that I've done it right! 😅

Reply:
Perfect, thanks @lfdebrux!